frontend/app/helpers/useLoginValidation.ts

```diff
@@ -0,0 +1,73 @@
+import { useState } from "react"
+
```

Comment:
I think it would be better if we not create a custom hook. I would suggest instead to create a `login.helper.ts` file in `helpers` folder and move all the functions here.

Reply:
i have change the file from custome hook to component